src/models/GenericOrgan.xl

```diff
@@ -0,0 +1,23 @@
+package models;
+
+import functions.FunctionCollector;
+
+public abstract class GenericOrgan {
+    private FunctionCollector collector;
```

Comment:
Could also try to remove from here and define it as static in inheriting classes (Leaf and Fruit)